settings.gradle

```diff
@@ -57,7 +61,7 @@ if (localProperties != null) {
     if (androidComponentsLocalPath != null) {
         log("Enabling automatic publication of android-components from: $androidComponentsLocalPath")
         log("Determining if android-components are up-to-date...")
-        def compileAcCmd = ["${androidComponentsLocalPath}/gradlew", "compileReleaseKotlin"]
+        def compileAcCmd = [gradlew, "compileReleaseKotlin"]
```

Comment:
Actually looking at it looks like neither needs the full path. It works specifying the full path to the command we are using. But the "working path" is passed to runCmd anyways. I just tried it not specifying the full path in either place and works fine on Windows anyways.

This was a straight squash of what's in Fenix now. I don't have any way of testing that either way would break anything on Mac builds which I'm assuming most are working with so I didn't change anything from what was already accepted there.

Reply:
Fair point, thanks for clarifying. 🙂